src/simulator/dice.py

```diff
@@ -10,9 +10,7 @@ class DiceValues(Enum):
     Skull = auto()
     Sword = auto()
 
-@dataclass
+
+@dataclass(unsafe_hash=True)
```

Comment:
Is there any reason to add unsfe_hash=True?

Reply:
In order to use Counter, Dice has to be hashable. By default, dataclass is hashable only if it imutable.